src/app/dataValidator.jsx

```diff
@@ -0,0 +1,101 @@
+const yaml = require('yaml-js'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
This file should be a `js` file, rather than a `jsx` file. Same with the test file below.

Reply:
Good point, will change